src/linter/linterReporting.ts

```diff
@@ -0,0 +1,44 @@
+export const MESSAGES = {
+	SHORT__NO_DIRECT_DATATYPE_ACCESS: "Deprecated access to DataType ('{0}')",
+	DETAILS__NO_DIRECT_DATATYPE_ACCESS: "DataType can be accessed only via the static " +
+	"DataType.getType(\"{0}\") method. " +
```

Comment:
The full solution is more complex than just calling `DataType.getType`, right?
For enums it is easier, but there the messageDetails just refer to the dev guide page, which I think is good as only the page can explain this with all details.
So I would suggest to do the same for DataTypes, just mention the dev guide page for details.

Reply:
Done